### ISSUE TITLE
Triaging and Project Management

### ISSUE THREAD

**Issue by Gnuxie** [the-draupnir-project/Draupnir#188]
Going to try base labels on these blog posts and tailscale's tracker

https://lostgarden.home.blog/2008/05/20/improving-bug-triage-with-user-pain/
https://apenwarr.ca/log/20171213

**Comment by Mikaela** [comment]
Agreed, I am going by my intuition with my recent labeling spree and I don't know if it's understandable or even if I will understand it in the morning.
